plugins/scaffolder-backend/api-report.md

```diff
@@ -552,6 +559,8 @@ export interface TaskStore {
     }[];
   }>;
   // (undocumented)
+  recoverTasks?(options: TaskStoreRecoverTaskOptions): Promise<string[]>;
```

Comment:
Return type: `Promise<{ ids: string[]}>` so that we can extend it easily in the future.